### ISSUE TITLE
[Topic RD] All components - issues after ACC testing

### ISSUE THREAD

**Issue by TeodorTaushanov** [SAP/ui5-webcomponents#5827]
- [x] ui5-label:
 - Label.for=inputId - label is not read, when the input is focused if accessible-name-ref="labelId" is not set (NVDA and JAWS).
Can we do this automatically like in OpenUI5?

This HTML is read correctly:
```
<label for="input1">Enter something</label>
<input id="input1">
```
https://snippix.only.sap/snippets/72742

 - "*" is not read in Virtual Mode. Align it with OpenUi5
 - Required is not read - check the samples. Align it with OpenUi5?

https://codesandbox.io/s/ui5-dialog-accessible-name-forked-f11ceg

- [x] ui5-radio-button
 - remove aria-readonly for the root element (access continuum issue)
 - align it with the OpenUI5 RadioButton. Check all aria attributes - on root level and on the 'input' element.

- [x] ui5-side-navigation
 - In expanded mode, the "2 of 3" is not read when move with the arrow keys. It is read only the first time the focus goes there. Change it like in OpenUI5?
 - In collapsed mode the aria attributes should be aligned with the OpenUI5.

- [x] ui5-tabcontainer
 - Tab design (like design="Negative") is not read.
